Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json

```diff
@@ -0,0 +1,710 @@
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailbcc": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailbody": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailbodyformat": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailbodyhtml": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailcc": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailclientname": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailfrom": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailheaders": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailhtml": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailinreplyto": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailkeywords": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailmessageid": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailreceived": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailreplyto": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailreturnpath": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailsenderip": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailsource": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailsubject": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailto": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
```

Comment:
Why do we need to extract all indicator types from Email To?

Reply:
email, domain, account no?

Reply:
Yes you're right actually. But isn't email = account? `Account.Email.Address`